tests/unit/fem/test_pa_coeff.cpp

```diff
@@ -144,4 +156,240 @@ TEST_CASE("pa_coeff")
+               if (integrator < 2)
+               {
+                  paform.AddDomainIntegrator(new VectorFEMassIntegrator(*coeff));
+               }
+               if (integrator > 0)
+               {
+                  paform.AddDomainIntegrator(new CurlCurlIntegrator(*curlCoeff));
+               }
+               paform.Assemble();
+               OperatorHandle paopr;
+               paform.FormSystemMatrix(ess_tdof_list, paopr);
+
+               BilinearForm assemblyform(&ND_fespace);
+               if (integrator < 2)
+               {
+                  assemblyform.AddDomainIntegrator(
+                     new VectorFEMassIntegrator(*coeff));
+               }
+               if (integrator > 0)
+               {
+                  assemblyform.AddDomainIntegrator(new CurlCurlIntegrator(*curlCoeff));
+               }
+               assemblyform.SetDiagonalPolicy(Matrix::DIAG_ONE);
+               assemblyform.Assemble();
+               assemblyform.Finalize();
+	       SparseMatrix A_explicit;
+	       assemblyform.FormSystemMatrix(ess_tdof_list, A_explicit);
+
+               Vector xin(ND_fespace.GetTrueVSize());
+               xin.Randomize();
+               Vector y_mat(xin);
+               y_mat = 0.0;
+               Vector y_assembly(xin);
+               y_assembly = 0.0;
+               Vector y_pa(xin);
+               y_pa = 0.0;
+
+               paopr->Mult(xin, y_pa);
+               assemblyform.Mult(xin, y_assembly);
+               A_explicit.Mult(xin, y_mat);
+
+               y_pa -= y_mat;
+               double pa_error = y_pa.Norml2();
+               std::cout << "  order: " << order
+                         << ", pa error norm: " << pa_error << std::endl;
+               REQUIRE(pa_error < 1.e-11);
+
+               y_assembly -= y_mat;
+               double assembly_error = y_assembly.Norml2();
+               std::cout << "  order: " << order
+                         << ", assembly error norm: " << assembly_error
+                         << std::endl;
+               REQUIRE(assembly_error < 1.e-12);
+
+               delete ND_fec;
+            }
+         }
+	 
+	 delete coeff;
+	 delete curlCoeff;
+      }
+      
+      delete mesh;      
+   }
+}
+
+TEST_CASE("Hcurl H1 mixed pa_coeff")
+{
+   for (dimension = 2; dimension < 4; ++dimension)
+   {
+      Mesh* mesh;
+      const int ne = 2;
+      if (dimension == 2)
+      {
+	 mesh = new Mesh(ne, ne, Element::QUADRILATERAL, 1, 1.0, 1.0);
+      }
+      else
+      {
+	 mesh = new Mesh(ne, ne, ne, Element::HEXAHEDRON, 1, 1.0, 1.0, 1.0);
+      }
+
+      for (int coeffType = 0; coeffType < 2; ++coeffType)
+      {
+	 Coefficient* coeff = nullptr;
+	 if (coeffType == 0)
+	 {
+	    coeff = new ConstantCoefficient(12.34);
+	 }
+	 else if (coeffType == 1)
+	 {
+	    coeff = new FunctionCoefficient(&coeffFunction);
+	 }
+	 
+         for (int integrator = 0; integrator < 1; ++integrator)
```

Comment:
Do you have plans to add another integrator option?  If not, perhaps we should remove this loop.

Reply:
I do not plan to add another integrator, so this loop could be removed, or it could be kept in case someone adds one later. Someone might add ND-L2, ND-RT, etc.

Reply:
Ok, let's just add a comment suggesting these additions as similar PA integrators become available.